.travis.yml

```diff
@@ -1,17 +1,20 @@
 language: go
 
-dist: bionic
 sudo: required
 
 go: "1.15"
 
-arch:
-  - amd64
-  - arm64
 
 jobs:
+  include:
+    - arch: amd64
+      dist: bionic
```

Comment:
I just noticed recentlly, that the cilium runtime is with focal. Not sure if we can upgrade amd64 to dist to focal here.

```shell script
$ docker run -it cilium/cilium:latest cat /etc/os-release
NAME="Ubuntu"
VERSION="20.04.1 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.1 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```

Reply:
and vagrant dev VM is with bionic right now as well :thinking:

Reply:
Focal is available for Amd64 job; 
Good comments, Thanks.